.editorconfig

```diff
@@ -1,3 +1,8 @@
+;
+; File managed by ModuleSync - Do Not Edit
+;
+; Additional Makefiles can be added to `.sync.yml` in 'Makefile.includes'
```

Comment:
Looks like there's a bug in the modulesync template somewhere, I'm pretty sure this comment doesn't make any sense.

Reply:
Opened a PR https://github.com/projectsyn/modulesync-control/pull/28

Going to merge this anyway to not get mixups in #6 .